api/handlers.go

```diff
@@ -51,6 +52,32 @@ type ServerImplementation struct {
 func validateTransactionFilter(filter *idb.TransactionFilter) error {
 	var errorArr = make([]string, 0)
 
+	// Round or application-id or asset-id is greater than math.MaxInt64
+	if filter.MinRound > math.MaxInt64 || filter.MaxRound > math.MaxInt64 ||
+		(filter.Round != nil && *filter.Round > math.MaxInt64) ||
+		filter.AssetID > math.MaxInt64 || filter.ApplicationID > math.MaxInt64 {
+		errorArr = append(errorArr, errValueExceedingInt64)
+	}
+
+	// offset > math.MaxInt64
+	if (filter.Offset != nil && *filter.Offset > math.MaxInt64) ||
+		(filter.OffsetLT != nil && *filter.OffsetLT > math.MaxInt64) ||
+		(filter.OffsetGT != nil && *filter.OffsetGT > math.MaxInt64) {
+		errorArr = append(errorArr, errValueExceedingInt64)
+	}
+
+	// algos > math.MaxInt64
+	if (filter.AlgosLT != nil && *filter.AlgosLT > math.MaxInt64) ||
+		(filter.AlgosGT != nil && *filter.AlgosGT > math.MaxInt64) {
+		errorArr = append(errorArr, errValueExceedingInt64)
+	}
+
+	// effectiveAmount > math.MaxInt64
+	if (filter.EffectiveAmountLT != nil && *filter.EffectiveAmountLT > math.MaxInt64) ||
```

Comment:
Should I update these from *uint64 to *int64 to match the table schema? I think we remove these checks if we update the var types.